final/server/codegen.ts

```diff
@@ -9,6 +9,7 @@ const config: CodegenConfig = {
         contextType: "./context#DataSourceContext",
         mappers: {
           Track: "./models#TrackModel",
+          Author: "./models#AuthorModel"
```

Comment:
Fewer updates here than on other PRs because we don't add the server codegen until this course; therefore there shouldn't be anything in place in the starter code folder